lua/keymappings.lua

```diff
@@ -1,73 +1,76 @@
-local function register_mappings(mappings, default_options)
-  for mode, mode_mappings in pairs(mappings) do
-    for _, mapping in pairs(mode_mappings) do
-      local options = #mapping == 3 and table.remove(mapping) or default_options
-      local prefix, cmd = unpack(mapping)
-      pcall(vim.api.nvim_set_keymap, mode, prefix, cmd, options)
-    end
-  end
-end
+local lv_utils = require "lv-utils"
+
+local opts = {
+  nnoremap = { noremap = true, silent = true },
```

Comment:
Now that I reread it, the key naming is quite weird, it should probably be {normal, insert, visual ect ...)
Feel free to leave it like this if you don't want to bother

Reply:
It's just that `O.keys.insert` would be kinda confusing, and `O.keys.inoremap` also looks weird. So I opted for the most readable one imo.

Keep in mind that the user might want to modify them from `lv-config`.